### PR TITLE
Update OrbitalColony.netkan

### PR DESCRIPTION
**Pull request by Real-Gecko**
As this mod is just a set of parts that use stock functionality it's version independent. Module Manager is recommended for integration with other mods, but not a requirement.

### CODE REVIEW

**Comment by Real-Gecko** [comment]
Hmm, I cannot understand what went wrong.

**Comment by HebaruSan** [comment]
Running it locally, it's confused by something about the KSP versions:

```
1601 [1] FATAL CKAN.NetKAN.Program (null) - One of the identified items was in an invalid format.
1602 [1] FATAL CKAN.NetKAN.Program (null) -   at CKAN.Versioning.KspVersion.Parse (System.String input) [0x0002c] in <a83a0b4a9df744b19439a2cf381b428e>:0 
  at CKAN.NetKAN.Transformers.AvcTransformer.Transform (CKAN.NetKAN.Model.Metadata metadata) [0x001d0] in <a83a0b4a9df744b19439a2cf381b428e>:0 
  at CKAN.NetKAN.Transformers.NetkanTransformer.<Transform>m__0 (CKAN.NetKAN.Model.Metadata transformedMetadata, CKAN.NetKAN.Transformers.ITransformer transformer) [0x00000] in <a83a0b4a9df744b19439a2cf381b428e>:0 
  at System.Linq.Enumerable.Aggregate[TSource,TAccumulate] (System.Collections.Generic.IEnumerable`1[T] source, TAccumulate seed, System.Func`3[T1,T2,TResult] func) [0x00037] in <63992662b765477a898ef49cdcc99ee2>:0 
  at CKAN.NetKAN.Transformers.NetkanTransformer.Transform (CKAN.NetKAN.Model.Metadata metadata) [0x00001] in <a83a0b4a9df744b19439a2cf381b428e>:0 
  at CKAN.NetKAN.Program.Main (System.String[] args) [0x000c5] in <a83a0b4a9df744b19439a2cf381b428e>:0 
```

It looks like there may be a bug in netkan.exe relating to the "any" string (which [the spec](https://github.com/KSP-CKAN/CKAN/blob/master/Spec.md#ksp_version) explicitly says is allowed).

File: NetKAN/OrbitalColony.netkan [hunk 1]
```diff
@@ -4,7 +4,8 @@
     "license": "CC-BY-NC-4.0",
     "$kref": "#/ckan/spacedock/1559",
     "$vref": "#/ckan/ksp-avc",
```
Comment (Olympic1) [inline NetKAN/OrbitalColony.netkan]: Remove the vref. `vref` and `any` dont work together.

**Comment by Real-Gecko** [comment]
Hmmm, removed $vref, but it did not help.

**Comment by Olympic1** [comment]
It installs fine now, but there are connection problems with GitHub

**Comment by Real-Gecko** [comment]
Alrighty, then everyting seems to be fine with actual .netkan file, am I right?

**Comment by Olympic1** [comment]
Yes, merging

**Comment by Real-Gecko** [comment]
Thanks!

**Pull request merged**